lib/sigfoxServer.js

```diff
@@ -101,11 +101,33 @@ function start(newConfig, callback) {
     iotServer.server.listen(iotServer.app.get('port'), iotServer.app.get('host'), callback);
 }
 
+/**
+ * Stop the current IoT Agent instance.
+ */
 function stop(callback) {
     config.getLogger().info(context, 'Stopping Sigfox server');
 
     iotServer.server.close(callback);
 }
 
+/**
+ * Shuts down the IoT Agent in a graceful manner
+ *
+ */
+function handleShutdown(signal) {
+    config.getLogger().fatal(context, 'Received %s, starting shutdown processs', signal);
```

Comment:
Should be info, as in PR https://github.com/telefonicaid/iotagent-json/pull/606

Reply:
I'd already done that - just didn't push. Fixed 6f9106f